src/command_modules/azure-cli-reservations/azure/cli/command_modules/reservations/tests/test_reservations_commands.py

```diff
@@ -0,0 +1,132 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest
+
+
+class AzureReservationsTests(ScenarioTest):
+
+    def _validate_reservation_order(self, reservation_order):
+        self.assertIsNotNone(reservation_order)
+        self.assertTrue(reservation_order['etag'])
+        self.assertTrue(reservation_order['id'])
+        self.assertTrue(reservation_order['name'])
+        self.assertTrue(reservation_order['reservationsProperty'])
+        self.assertTrue(reservation_order['displayName'])
+
+    def _validate_reservation(self, reservation):
+        self.assertIsNotNone(reservation)
+        self.assertTrue(reservation['id'])
+        self.assertTrue(reservation['name'])
+        self.assertTrue(reservation['kind'])
+        self.assertTrue(reservation['sku'])
+        self.assertTrue(reservation['properties'])
+        self.assertTrue(reservation['type'])
+
+    def test_get_applied_reservation_order_ids(self):
+        result = self.cmd('az reservations reservation-order-id list --subscription-id 00000000-0000-0000-0000-000000000000').get_output_in_json()
+        for order_id in result['reservationOrderIds']['value']:
+            self.assertTrue('/providers/Microsoft.Capacity/reservationorders/' in order_id)
```

Comment:
Could use self.assertIn. This would give a clearer error message on failure.
Similar elsewhere.
https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIn